ui/pom.xml

```diff
@@ -118,6 +118,21 @@
             </plugin>
 
             <plugin>
+                <groupId>org.wildfly.plugins</groupId>
+                <artifactId>wildfly-maven-plugin</artifactId>
+                <version>1.1.0.Alpha7</version>
```

Comment:
Is the version still necessary here (since the parent pom has it in pluginManagement already)?